AdMobAdapter/src/main/java/com/chartboost/mediation/admobadapter/AdMobAdapter.kt

```diff
@@ -154,18 +154,16 @@ class AdMobAdapter : PartnerAdapter {
     override suspend fun setUp(
         context: Context,
         partnerConfiguration: PartnerConfiguration,
-    ): Result<Unit> {
+    ): Result<Unit> = withContext(IO) {
         PartnerLogController.log(SETUP_STARTED)
 
-        withContext(IO) {
             // Since Chartboost Mediation is the mediator, no need to initialize AdMob's partner SDKs.
             // https://developers.google.com/android/reference/com/google/android/gms/ads/MobileAds?hl=en#disableMediationAdapterInitialization(android.content.Context)
             //
             // There have been known ANRs when calling disableMediationAdapterInitialization() on the main thread.
             MobileAds.disableMediationAdapterInitialization(context)
-        }
 
-        return suspendCancellableCoroutine { continuation ->
+        return@withContext suspendCancellableCoroutine { continuation ->
```

Comment:
not sure you need the return statement here since the last item in a {} should be the return value, but it's not wrong to have this here.
can you port this change to the 5.x adapters, too? thanks

Reply:
I had an error if I didn't have it.

Reply:
Oh, but if I completely remove the return there isn't an error.  I'll take care of removing those soon.